content/docs/tasks/security/authz-tcp/index.md

```diff
@@ -188,13 +188,26 @@ to access the MongoDB service.
 
 1. To confirm the MongoDB service can only be accessed by service account `bookinfo-ratings-v2`:
 
-    Run the following command to re-deploy the `v2` of `ratings` with service account `default`:
+    Run the following command to delete the `ratings` with service account `bookinfo-ratings-v2`:
```

Comment:
I presume here and below, you mean 'the ratings service'?

Reply:
It mean the `ratings` deployment, updated the doc to add this.